### PR TITLE
 New sidebar navigation component

### PR DESCRIPTION
**Pull request by stjanilofts**
A new sidebar navigation component for recursive menu nav.

## Screenshots / Gifs

image
Screen Recording 2020-12-03 at 21 44 08

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/island-ui/core/src/lib/Navigation/Navigation.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+
+describe('Navigation', () => {
+  it('should render successfully', () => {
+    const { baseElement } = render(<div>ok</div>)
```
Comment (barabrian) [inline libs/island-ui/core/src/lib/Navigation/Navigation.spec.tsx:6]: this is questionable 🧐 
maybe remove useless test or try to render the navigation 🙂

**Review by barabrian — CHANGES_REQUESTED** [review]
LGTM, one minor comment :)

File: libs/island-ui/core/src/lib/Navigation/Navigation.spec.tsx (hunk 1 above)
Comment (siggakogk) [reply libs/island-ui/core/src/lib/Navigation/Navigation.spec.tsx:6]: lol didn't notice this! i will remove it

**Review by barabrian — APPROVED** [review]
lgtm

**Pull request merged**